backend/src/main/java/ubb/courses/backend/DTOs/CourseDTO.java

```diff
@@ -0,0 +1,60 @@
+package ubb.courses.backend.DTOs;
+
+import ubb.courses.backend.models.Course;
+
+import java.util.Date;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class CourseDTO extends BaseDTO<Course, Long> {
```

Comment:
The DTO should not be created into this pr, there is a separate feature for that. They should be implemented/updated when the create read update delete controller methods are created.

Reply:
The reason for this it's because the DTO for creating might be updated/different than the one for updating a course